app/models/mixins/relationship_mixin.rb

```diff
@@ -134,6 +134,16 @@ def parent_ids(*args)
     Relationship.resource_pairs(parent_rels(*args))
   end
 
+  # return a scope of all of the parents of this record
+  def parents_scope(*args)
```

Comment:
If we want parents to be a scope, can we just changes parents to do this directly?